src/fragments/lib/logging/ios/setup_logging/20_initialize_with_config.mdx

```diff
@@ -0,0 +1,43 @@
+In your mobile app, create and add a `amplifyconfiguration_logging.json` to the same location as the `amplifyconfiguration.json` file.  Ensure that the file is included in the `Copy Bundle Resources` build phase.
+
+The `<log-group-name>` and `region` is the value you specified in the CDK construct as part of provisioning your AWS backend resources.  The configuration file is the data source for the logging plugin to know where, when and what to send logs.  The example below configures the logging plugin to automatically send all logs at log level `ERROR` at 60 seconds interval and store logs locally up to 1MB.
+
+```json
+{
+    "awsCloudWatchLoggingPlugin": {
+        "enable": true,
+        "logGroupName": "<log-group-name>",
+        "region": "<region>",
+        "localStoreMaxSizeInMB": 1,
+        "flushIntervalInSeconds": 60,
+        "loggingConstraints": {
+            "defaultLogLevel": "ERROR"
+        }
+    }
+}
+```
+
+To use the Amplify Logger and Amplify Auth categories in your app, you need to create and configure their corresponding plugins by calling the `Amplify.add(plugin:)` and `Amplify.configure()` methods.
+
+Add the following imports to the top of your main `App` file:
+
+```swift
+import Amplify
+import AWSCognitoAuthPlugin
+import AWSCloudWatchLoggingPlugin
+```
+
+Add the following code to its initializer. If there is none, you can create a default `init`:
+
+```swift
+init() {
+    do {
+        try Amplify.add(plugin: AWSCognitoAuthPlugin())
+        try Amplify.add(plugin: AWSCloudWatchLoggingPlugin())
```

Comment:
it is not clear to me where I pass the logging configuration file. Does that happen by default without needing to specify a path?

Reply:
Yes, the configuration file is automatically looked up.  I've added additional text to describe this behavior.

Reply:
Reads really well now!